lib/aggregate/aggregate_store.rb

```diff
@@ -71,15 +73,45 @@ def self.included(model_class)
       model_class.extend ClassMethods
     end
 
+    def save
```

Comment:
We should have the `active-record` callbacks that we make use of, no? Can't we use the `after_save` hook here to handle this?

Reply:
I don't quite follow